src/cmd/src/standalone.rs

```diff
@@ -416,4 +437,37 @@ mod tests {
         let toml_string = toml::to_string(&opts).unwrap();
         let _parsed: StandaloneOptions = toml::from_str(&toml_string).unwrap();
     }
+
+    #[test]
+    fn test_logging_options_toml() {
+        let cmd = StartCommand {
+            http_addr: None,
+            rpc_addr: None,
+            mysql_addr: None,
+            prom_addr: None,
+            postgres_addr: None,
+            opentsdb_addr: None,
+            config_file: Some(format!(
+                "{}/../../config/standalone.example.toml",
```

Comment:
suggestion: You could construct a string literal instead of using the example.toml

Example:
https://github.com/GreptimeTeam/greptimedb/blob/eb50cee6019ae1a45d65213cdec5b5614a0f4622/src/cmd/src/datanode.rs#L192-L197